src/lib/units/offset.js

```diff
@@ -161,11 +161,10 @@ export function setOffsetToLocal (keepLocalTime) {
 }
 
 export function setOffsetToParsedOffset () {
-    if (this._tzm) {
+    if (this._tzm === 0 || this._tzm) {
```

Comment:
Seems pretty legible to me - I don't quite follow.  Would you provide an example?  Your initial patch was failing when `this._tzm`was `0`. ( Unless im wrong :] ) Are you proposing we check if its a number OR a non empty string?  Checking if its a number will fail on `NaN` so we would have to add the clause for that as well.